whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+\end{figure}
+
+\begin{figure}[tbh!]
+\plottwo{figs/WL/minion_1016_Kuiper_ParallacticAngle_propID_54_and_i_HEAL_SkyMap.pdf}
+        {figs/WL/minion_1016_Kuiper_ParallacticAngle_u_g_r_i_z_y_propID_54_HEAL_ComboHistogram.pdf}
+\caption{Same as Fig. \ref{fig:WL_Kuiper_rotSkyPos}, but for the parallactic
+    angle instead of rotSkyPos.}
+\label{fig:WL_Kuiper_ParallacticAngle}
+\end{figure}
+
+
+\subsection{Ancilliary data}
+
+Optimal observing strategy relies on the use of all relevant data, with current
+and historical coverage per field.  Ancilliary data, and auxilliary uses of the
+science exposures, play an important role informing next-field strategy.  We can
+use large scale patterns of distortions of the PSF over the 20,000 stars per
+exposure for PSF regularization in the per-CCD PSF fitting.  In the per CCD
+fits, there is a benefit to setting aside some stars for validation tests of PSF
+extrapolation.  These data may be used as a metric for image quality and thus
+the ranked value of an exposure for shear systematics removal.  Fields with poor
+image quality rise to the top of the priority list for re-observing at that
+camera angle.  In addition to using all the stars in a given visit, there is
+useful information in the wavefront sensors and the guide CCDs that may be used
+to regularize the PSF reconstruction in a visit.  We might read out guider CCDs
+in different ways to better monitor the atmosphere.
+
+
+\subsection{Deep vs Wide}
+
+As outlined above, many revisits to each field spanning many RotSkyPos angles
+aids the suppression of shear systematics.  For a given per-visit exposure time
+this leads to a deep survey.  There are several advantages to a deep survey over
+a shallow-wide survey for weak lensing science.  A strategy question for LSST is
+whether to go wide first and then deep, or the reverse.
+
+There are actually several drivers for depth over area, given fixed observing
+time and camera+telescope etendue.  A deep observing strategy maximizes the
+cosmological signal-to-noise ratio either by maximizing the signal or minimizing
+the noise.
+
+First, it boosts the amplitude of the cosmic shear signals due to the increased
+amplitude of the lensing kernel.  Given the same lens mass, the amplitude of
+lensing signals is a simple function of the two distances: observer-to-lens
+versus lens-to-source. For example, when a lens is at z = 0.5, the shear for a
+source at z = 1.1 is nearly twice that at z = 0.7, leading to a factor of 4
+ratio in correlation function amplitudes.  Thus, a deep survey can take
+advantage of the geometric effect of gravitational lensing more efficiently.
+
+Second, it enables a longer redshift baseline to break parameter degeneracies.
+For example, shallower surveys with wider area are not efficient in shrinking
+the length of the ``banana'' in the $\Omega_M$-$\sigma_8$ plane because of the
+$\sigma_8 \Omega_M^\alpha = \mathrm{const}$ degeneracy.  The deep strategy
```

Comment:
True.  But LSST will be a survey of a far larger volume, so on balance 
depth wins for WL.   That was the main driver for LSST in the first 
place (back when we were deciding the aperture).

On 6/29/16 7:38 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69061377>:
>
>> +time and camera+telescope etendue.  A deep observing strategy maximizes the
>> +cosmological signal-to-noise ratio either by maximizing the signal or minimizing
>> +the noise.
>> +
>> +First, it boosts the amplitude of the cosmic shear signals due to the increased
>> +amplitude of the lensing kernel.  Given the same lens mass, the amplitude of
>> +lensing signals is a simple function of the two distances: observer-to-lens
>> +versus lens-to-source. For example, when a lens is at z = 0.5, the shear for a
>> +source at z = 1.1 is nearly twice that at z = 0.7, leading to a factor of 4
>> +ratio in correlation function amplitudes.  Thus, a deep survey can take
>> +advantage of the geometric effect of gravitational lensing more efficiently.
>> +
>> +Second, it enables a longer redshift baseline to break parameter degeneracies.
>> +For example, shallower surveys with wider area are not efficient in shrinking
>> +the length of the ``banana'' in the $\Omega_M$-$\sigma_8$ plane because of the
>> +$\sigma_8 \Omega_M^\alpha = \mathrm{const}$ degeneracy.  The deep strategy
>
> They can, however, place better constraints on w at low redshift, so if
> that's what you care about, then shallower surveys win. (Hence DES is
> better at constraining w at low redshift than HSC, and then above some
> redshift HSC becomes more powerful because it's deeper.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69061377>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6YsSK5q2tX6eVaccBha2NdXtOhmuGks5qQywpgaJpZM4I9hd->.
>